thirdeye-server/src/main/java/ai/startree/thirdeye/auth/basic/BasicAuthConfiguration.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package ai.startree.thirdeye.auth.basic;
+
+import java.util.List;
+
+
+public class BasicAuthConfiguration {
+
+  private boolean enabled;
```

Comment:
default to `false`. 

Although, by default primitive bools are false, let's call it out explicitly here

Reply:
Done